src/app/dataset-view/dataset-view.interface.ts

```diff
@@ -6,3 +18,21 @@ export enum DatasetViewTypeEnum {
     discussions = "discussions",
     history = "history",
 }
+
+export interface DatasetViewContentInterface {
+    isTableHeader: boolean;
+    displayedColumns?: any[];
+    datasetDataSource?: SearchHistoryInterface[];
```

Comment:
Please double-check correspondence of variable names and data types here.
Maybe I'm missing something, but why `datasetDataSource` is `SearchHistoryInterface`, and `datasetOverviewSource` is `SearchHistoryInterface`, and `datasetHistorySource` is `MetadataBlockExtended`. Something must be wrong here, either types or names.